### PR TITLE
Upload wheels on release

### PR DESCRIPTION
**Pull request by jd**
This leverages scripts/build-dist to build the wheel and upload them to PyPI
when it's time to release.

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -62,21 +62,17 @@ namespace :pypi do
 
   task :build => :clean do
     puts "building release in #{RELEASE_DIR}"
-    # TODO: Use `scripts/build-dist` instead to build sdist and wheels
     sh "python setup.py -q sdist -d #{RELEASE_DIR}"
```
Comment (brettlangdon) [inline Rakefile]: `scripts/build-dist` will run this for us.

**Comment by brettlangdon** [comment]
I wonder what the best way to test this is before doing it on a real release.

**Comment by jd** [comment]
I tested it locally and it worked fine. I wondered if we can run it on CircleCI but I doubt it.
I'm good with that and fixing it at release time if needed.

**Comment by brettlangdon** [comment]
cool, and yeah, worse case we comment it out temporarily if it gives us issues, etc.

yeah, CircleCI is tough... wouldn't mind releases being in there.

also, please hold this until the next release is out, don't want to surprise @majorgreys while he is mid-release.

**Comment by jd** [comment]
removed sdist from Rakefile

File: Rakefile [hunk 2]
```diff
@@ -62,21 +62,16 @@ namespace :pypi do
...
-    elsif builds.length > 1
-        fail "multiple builds found in #{RELEASE_DIR}"
     end
 
-    build = "#{RELEASE_DIR}/#{builds[0]}"
-
-    puts "uploading #{build}"
-    sh "twine upload #{build}"
+    puts "uploading #{RELEASE_DIR}/*"
+    sh "twine upload #{RELEASE_DIR}/*"
```
Comment (jd) [inline Rakefile]: Do we need to make sure that the RELEASE_DIR was empty before building and uploading?
I'm afraid we'd upload old/stale build.

**Comment by brettlangdon** [comment]
Yes we do need to make sure it is empty.

**Comment by jd** [comment]
I just noticed the dist directory is cleaned by `build-dist`.
https://github.com/DataDog/dd-trace-py/blob/master/scripts/build-dist#L12
So we should be good.

**Pull request merged**